.github/workflows/test_dockerfile.yml

```diff
@@ -0,0 +1,11 @@
+name: Test building the Dockerfile
+
+on:
+  workflow_dispatch:
```

Comment:
Well, I was thinking about that, but that's gonna take a bit some time to check.
I also would like to add a "install nodejs, rustup and opam in the system package manager and then run the setup.sh" GH action

But this and the docker file test are kinda slow, and I'm worried this will be too bothering to have this in each commit of each PR...

Was thinking maybe we can use https://github.com/Khan/pull-request-comment-trigger, so that in PR, just before merging, we ask the GH actions to check extra install-related things that are a bit long to run. What do you think?

Reply:
Yeah, using comment triggers is a good idea in general. We could also go a step further and use something like [bors](https://bors.tech/).
The other option is to have a nightly job for the long running tasks.
Either way, I'm fine with getting this in as is and adding other triggers later.

Reply:
Oh I did not know about bors! That seems to me like a really nice workflow, I really like it!
Just opened an issue for adding support!

Reply:
merging this PR thus